src/WireMock.Net/ResponseBuilders/Response.cs

```diff
@@ -22,10 +23,31 @@ namespace WireMock.ResponseBuilders
     /// </summary>
     public partial class Response : IResponseBuilder
     {
+        private static readonly ThreadLocal<Random> Random = new ThreadLocal<Random>(() => new Random(DateTime.UtcNow.Millisecond));
+
+        private int _minDelayMilliseconds;
+        private int _maxDelayMilliseconds;
+        private bool _hasRandomDelay;
+
+        private TimeSpan? _delay;
+
         /// <summary>
         /// The delay
         /// </summary>

```

Comment:
Suggestion
Maybe better to remove this property.

And add two new properties:
- `MinimumDelay`
- `MaximumDelay`

And in case only the old situation is needed, just use one, like MinimumDelay.
And only calculate a random value is the MinimumDelay and MaximumDelay are defined

?